packages/react-component-library/playwright.config.ts

```diff
@@ -0,0 +1,123 @@
+import { PlaywrightTestConfig, devices } from '@playwright/test'
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config: PlaywrightTestConfig = {
+  testDir: './playwright/tests',
+
+  globalSetup: require.resolve('./playwright/tests/globalSetup'),
+
+  /* Maximum time one test can run for. */
+  timeout: 30 * 1000,
+
+  expect: {
+    /**
+     * Maximum time expect() should wait for the condition to be met.
+     * For example in `await expect(locator).toHaveText();`
+     */
+    timeout: 5000,
+  },
+
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+
+  maxFailures: process.env.CI ? 10 : undefined,
+
+  /* Set to 1 locally to enable trace recording. */
+  retries: process.env.CI ? 1 : 0,
+
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: [
+    [process.env.CI ? 'github' : 'line'],
+    [
+      'html',
+      {
+        outputFolder: './playwright/test-results/html/',
+        open: process.env.PWDEBUG ? 'never' : 'on-failure',
+      },
+    ],
+  ],
+
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Maximum time each action such as `click()` can take. Defaults to 0 (no limit). */
+    actionTimeout: process.env.CI ? 60_000 : 10_000,
+
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    // baseURL: 'http://localhost:6007',
+
+    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+    trace: 'on-first-retry',
+
+    screenshot: 'only-on-failure',
+  },
+
+  /* Configure projects for major browsers */
+  projects: [
+    {
+      name: 'chromium',
+
+      /* Project-specific settings. */
+      use: {
+        ...devices['Desktop Chrome'],
+      },
+    },
+
+    {
+      name: 'firefox',
+      use: {
+        ...devices['Desktop Firefox'],
+      },
+    },
+
+    // {
```

Comment:
Webkit is supported as well – turned it off as I'm not sure it's useful for us (and it was a bit more faff to get working on the latest Ubuntu).